pkg/installer/uninstall.go

```diff
@@ -100,6 +100,13 @@ func (in *Installer) uninstallStorageOS(uninstallConfig apiv1.Uninstall) error {
 		return err
 	}
 
+	// if this is not an upgrade, write manifests to disk before deletion
+	if !upgrade {
```

Comment:
Any reason not to backup the manifests even if it's an upgrade?  Just in case something goes wrong and the user needs to try and resolve it manually.  

Or is that done in `prepareForUpgrade()`?